{"name": "test_user_auth[dynamic_test_user5-case18]", "status": "passed", "steps": [{"name": "发送请求：'POST' '/auth/login'", "status": "passed", "attachments": [{"name": "请求信息", "source": "9ae38b7e-6f03-4eb1-b6f2-48d9201ea1aa-attachment.txt", "type": "text/plain"}, {"name": "请求头", "source": "78697264-52b2-413c-8982-f73d62e92f81-attachment.txt", "type": "text/plain"}, {"name": "请求参数", "source": "b7b2eac1-570d-4c08-8339-fd344b0d38a0-attachment.json", "type": "application/json"}, {"name": "响应信息", "source": "f12b21e9-a0e8-49fa-9ecd-8bdfe64a86c5-attachment.txt", "type": "text/plain"}, {"name": "响应体", "source": "0e00ca0d-c7b8-4ea7-876a-0f0e36e89fdb-attachment.json", "type": "application/json"}], "parameters": [{"name": "method", "value": "'POST'"}, {"name": "endpoint", "value": "'/auth/login'"}, {"name": "headers", "value": "{}"}, {"name": "params", "value": "{}"}, {"name": "json", "value": "{'username': 'admin1155', 'password': 'Wrong123'}"}, {"name": "files", "value": "None"}, {"name": "token", "value": "None"}], "start": 1754636643710, "stop": 1754636643848}, {"name": "验证响应：状态码403，消息'密码错误，还可重试4次'", "status": "passed", "parameters": [{"name": "response", "value": "<Response [403]>"}, {"name": "expected_code", "value": "403"}, {"name": "expected_success", "value": "False"}, {"name": "expected_message", "value": "'密码错误，还可重试4次'"}, {"name": "is_file", "value": "False"}], "start": 1754636643848, "stop": 1754636643848}], "attachments": [{"name": "log", "source": "a53f1fba-16d7-4545-b499-afa2a5fdb5de-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "9bc62079-3775-4b6c-9999-47cb6958e01e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "dynamic_test_user", "value": "{'case_id': 'auth_006', 'name': '用户名包含特殊符号', 'method': 'POST', 'url': '/auth/register', 'json': {'username': 'test@1', 'password': 'Test123456', 'email': 'test_c80iwa@example.com', 'verifyCode': '123'}, 'expected_code': 400, 'expected_success': False, 'expected_message': '用户名格式无效（仅支持中文、英文、数字、下划线）'}"}, {"name": "case", "value": "{'case_id': 'auth_019', 'name': '密码错误登录', 'method': 'POST', 'url': '/auth/login', 'json': {'username': 'admin1155', 'password': 'Wrong123'}, 'expected_code': 403, 'expected_success': False, 'expected_message': '密码错误，还可重试4次'}"}], "start": 1754636643709, "stop": 1754636643850, "uuid": "fa992210-71dd-4930-8bcd-2a956c24f224", "historyId": "271ef2c5dfd4209ad0708dbcff5b47de", "testCaseId": "de18b51a986ebd96b4485f2c6341275e", "fullName": "tests.test_user_auth#test_user_auth", "labels": [{"name": "feature", "value": "用户认证模块"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_auth"}, {"name": "host", "value": "CYJ"}, {"name": "thread", "value": "11184-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_auth"}], "titlePath": ["tests", "test_user_auth.py"]}